docs/dev/libs/cep.md

```diff
@@ -429,6 +429,41 @@ patternState.within(Time.seconds(10));
       {% endhighlight %}
           </td>
        </tr>
+       <tr>
+          <td><strong>Consecutive</strong></td>
+          <td>
+              <p>Works in conjunction with zeroOrMore, oneOrMore or times. Specifies that any not matching element breaks the loop.</p>
```

Comment:
1) Mention here that the default behavior is relaxed continuity. Same for Scala. Also the default behavior, I would also put it in the docs for `oneToMore`, `zeroToMore`, and `times` and have a forward reference to `consecutive`.
2) At the end where you have the example, you could also mention that in the case of the default, we would have ...
3) And also now it is an `SimpleCondition`, not a `FilterFunction` ;)